apollo-sqlite-cache/src/main/java/com/apollographql/apollo/cache/normalized/sql/SqlNormalizedCache.kt

```diff
@@ -0,0 +1,97 @@
+package com.apollographql.apollo.cache.normalized.sql
+
+import com.apollographql.apollo.cache.ApolloCacheHeaders.EVICT_AFTER_READ
+import com.apollographql.apollo.cache.CacheHeaders
+import com.apollographql.apollo.cache.normalized.CacheKey
+import com.apollographql.apollo.cache.normalized.NormalizedCache
+import com.apollographql.apollo.cache.normalized.Record
+import com.apollographql.apollo.cache.normalized.RecordFieldJsonAdapter
+import okio.IOException
+
+class SqlNormalizedCache(
+    private val recordFieldAdapter: RecordFieldJsonAdapter,
+    private val cacheQueries: CacheQueries
+) : NormalizedCache() {
```

Comment:
What about `SQLDelightNormalizedCache` ?

Reply:
That would be backward incompatible.

Reply:
I see. Since it had queries in the constructor, I assumed it was already backward incompatible but this is only created through `SqlNormalizedCacheFactory`, right ? Then mark the constructor internal maybe ?

Reply:
Yes. That I missed. Before the constructor was package private. It is not something users should be instantiating themselves. They should use the factory.

But they may assign `SqlNormalizedCache` to somewhere. The name is still public. I don't like it either :) but we need to keep it.